### PR TITLE
Modify the schema tool to add prefix

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-7606

Modify the schema tool to add the specified prefix to each namespace with `--prefix` option

### CODE REVIEW

File: tools/scalar-schema/src/scalar_schema/core.clj [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "The network topology strategy. SimpleStrategy or NetworkTopologyStrategy. This options is ignored when Cosmos DB and DynamoDB."]
    ["-D" "--delete-all" "All database will be deleted, if this is enabled."]
    [nil "--region REGION" "Region where the tool creates tables for DynamoDB"]
+   [nil "--prefix DATABASE_PREFIX" "Database prefix. The prefix will be added to all database."]
```
Comment (feeblefakie) [inline tools/scalar-schema/src/scalar_schema/core.clj:25]: I think it should be consistent with Scalar DB configuration?
```suggestion
   [nil "--prefix NAMESPACE_PREFIX" "Namespace prefix. The prefix is added to all the namespaces."]
```

**Review by feeblefakie — COMMENTED** [review]
Overall looking good! Left 1 suggestion.

File: tools/scalar-schema/src/scalar_schema/core.clj (hunk 1 above)
Comment (yito88) [reply tools/scalar-schema/src/scalar_schema/core.clj]: Thanks!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thanks.

Maybe, we need to use `namespace` consistently in the program at some later point.

**Pull request merged**